tools/go-cli/go-whisk-cli/commands/util.go

```diff
@@ -438,9 +438,14 @@ func logoText() string {
     return logo
 }
 
-func printJSON(v interface{}) {
+func printJSON(v interface{}, stream ...io.Writer) {
```

Comment:
@dubeejw - Yes it did, but after piping the output to the colorable stdout/stderr, the formatting is fine for me.  Would you please confirm it's behavior in your environment.  It works well in my native Windows and Vagrant Ubuntu VM.